apps/src/lib/kits/maker/boards/microBit/MicroBitConstants.js

```diff
@@ -34,6 +34,12 @@ export const MB_SENSOR_VARS = [
   'compass'
 ];
 
+// milliseconds between samples for sensors
+export const SAMPLE_RATE = 50;
+
+// Max number of milliseconds for which we store historical data
+export const MAX_SENSOR_BUFFER = 3000;
```

Comment:
It feels like this variable name is missing a word indicating what this number means.  How about MAX_SENSOR_BUFFER_SIZE or MAX_SENSOR_SAMPLE_COUNT or something like that?